### ISSUE TITLE
[Test] High contrast is enabled after running the accessibility.liveRegion.activityStatus.sendFailed.contrast test

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4569]
### Is it an issue related to Adaptive Cards?

No

### Is this an accessibility issue?

No

### What version of Web Chat are you using?

Unrelated

### Which distribution are you using Web Chat from?

Unrelated

### Which hosting environment does this issue primarily affect?

Others or unrelated

### Which area does this issue affect?

Others or unrelated

### Please describe the bug

When tests are run after `accessibility.liveRegion.activityStatus.sendFailed.contrast.html`, their snapshot test will fail because the mentioned test enabled `forced-colors`.

We should unset `forced-colors` before every test run.

### How to reproduce the issue?

Tests are run in parallel with 4 workers, to increase the likelihood of the issue, we tune it down to 1 worker.

1. Modify `package.json/scripts/docker` and change `--scale chrome=4` to `--scale chrome=1`
1. Run `npm test -- --maxWorkers=1 --testPath sendfailed`

### What do you expect?

All tests should pass.

### What actually happened?

Some tests fail because the snapshot is taken in high contrast mode.

### Additional context

This is related to #4537.
